### ISSUE TITLE
HTTP server is not starting

### ISSUE THREAD

**Issue by kartikraj** [insidesherpa/JPMC-tech-task-2-PY3#81]
![Annotation 2020-04-27 115708](https://user-images.githubusercontent.com/47671868/80340950-32a32c00-887f-11ea-9c2e-5742aa30b60f.jpg)

**Comment by kartikraj** [comment]
@jbf-insidesherpa  please look into it

**Comment by Inderjeet0003** [comment]
I am also facing the same issue kindly help.

**Comment by Abhishek-pandey-1998** [comment]
only type 'python datafeed/server3.py

**Comment by kartikraj** [comment]
> only type 'python datafeed/server3.py

No it gives an error

**Comment by kartikraj** [comment]
@insidesherpa please look into it

**Comment by 9916103020** [comment]
@kartikraj please try one more time running cmd as admin and running same command which you typed because there is no error in the command:)

**Comment by kartikraj** [comment]
> @kartikraj please try one more time running cmd as admin and running same command which you typed because there is no error in the command:)

Yes I  did try again now,, same problem ,there is something wrong . Should I  try running next commands?

**Comment by 9916103020** [comment]
@kartikraj have you installed dateutils??

**Comment by kartikraj** [comment]
> @kartikraj have you installed dateutils??

Pip --version command gives no error so it's installed

**Comment by Inderjeet0003** [comment]
How to get my frontend on localhost of my pc.

**Comment by Inderjeet0003** [comment]
> > only type 'python datafeed/server3.py
> 
> No it gives an error

no bro it worked for me.

**Comment by SiddhlodaAyyar** [comment]
Watch this video and if you still get the error comment down in the youtube section and we'll try to solve it.
https://youtu.be/wRAG9i8O_44
